docs/sources/logql/template_functions.md

```diff
@@ -23,6 +24,23 @@ Example:
 {{ .path | replace " " "_" | trunc 5 | upper }}
 ```
 
+## __line__
+
+This function return the current log line.
+
+> Added in Loki 2.5
```

Comment:
2 changes: 
1. grammar correction
2. now that we have versioned docs, the Loki version number in which this function was added is irrelevant. (A task for another PR: remove all the notices that say what version of Loki a function was added to.)

```suggestion
This function returns the current log line.

```